base/ca/src/com/netscape/ca/KeyRetrieverRunner.java

```diff
@@ -85,6 +87,12 @@ private boolean _run() {
         CAEngine engine = CAEngine.getInstance();
         EngineConfig cs = engine.getConfig();
 
+        try {
+            useOAEPKeyWrap = cs.getBoolean("ca.use_oaep_rsa_key_wrap",false);
+        } catch (EBaseException e1) {
+            useOAEPKeyWrap = false;
```

Comment:
I'd suggest removing the `try-catch` block or rethrow the exception as `RuntimeException`. If the param is not defined, it will default to `false`. If the param contains non-boolean value, it should fail so the admin can see and fix it. Discarding the exception can make it more difficult to troubleshoot.

Reply:
It might be better to move the parameter loading into the constructor so the code will fail before the thread starts.